package.json

```diff
@@ -63,7 +63,7 @@
     "test:python": "test ! $(find tests -name \"*.py\") || python setup.py test",
     "predist:web": "npm run build:web && npm run docs:web",
     "dist:web": "mkdirp dist && cd build && tar cvzf ../dist/tdp_core.tar.gz *",
-    "dist:python": "python setup.py sdist bdist_wheel",
```

Comment:
wheel is the newer one

Reply:
See also https://github.com/phovea/generator-phovea/issues/259